src/map.config.js

```diff
@@ -95,7 +95,7 @@ async function getServices () {
   }
   let url = 'config/webconfig.json'
   // TODO: how shall we configure this? Discuss with Peter
-  // url = 'config/webconfig-rws.json'
+  url = 'config/webconfig-rws.json'
```

Comment:
can you comment this back? We now have a public test url which should be available by default.